3rdparty/rainterface/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+add_library(rainterface
```

Comment:
![image](https://user-images.githubusercontent.com/24227051/229131661-b9bcb6bf-277b-4e37-8f77-e9c818758f40.png)

Mix of tabs and spaces?

Reply:
I was hoping nobody would notice :'(